core/mutex.c

```diff
@@ -35,34 +35,38 @@
 #define ENABLE_DEBUG    (0)
 #include "debug.h"
 
-#define MUTEX_LOCKED ((void*)-1)
+#define MUTEX_LOCKED ((void *)-1)
```

Comment:
Removed, but don't agree... Pointers should be spaced according to the CC, so why not their casts?